packages/deployer/internal/config-defaults.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
+  proxyContract: 'Proxy',
   paths: {
-    modules: 'contracts/modules',
     deployments: 'deployments',
+    modules: 'modules',
```

Comment:
To follow common practices with other Hardhat plugins, now the `modules` folder is configured from the `sources` folder, not the root.